backend/database.js

```diff
@@ -37,38 +39,53 @@ if (!fs.existsSync(dataFolder)) {
   fs.mkdirSync(dataFolder, { mode: 0o750 })
 }
 
+// Streams stored contract log entries since the given entry hash (inclusive!).
 sbp('sbp/selectors/register', {
-  'backend/db/streamEntriesAfter': async function (contractID: string, hash: string): Promise<*> {
+  'backend/db/streamEntriesAfter': async function (contractID: string, hash: string, requestedLimit: number = MAX_EVENTS_AFTER): Promise<*> {
+    const limit = Math.min(requestedLimit, process.env.MAX_EVENTS_BATCH_SIZE ?? 500)
```

Comment:
It seems to me like `MAX_EVENTS_AFTER` isn't really relevant anymore in the face of this `Math.min` and `process.env.MAX_EVENTS_BATCH_SIZE ?? 500`, as that will always be less than `Infinity`.

Reply:
`MAX_EVENTS_AFTER` is a client-side constraint, while `MAX_EVENTS_BATCH_SIZE` is a server-side constraint.

Reply:
Yes, we can remove it from the server.